lib/vyasa_web/components/source_content/chapters.ex

```diff
@@ -0,0 +1,73 @@
+defmodule VyasaWeb.Content.Chapters do
+  use VyasaWeb, :live_component
+
+  @impl true
+  def update(params, socket) do
+    {
+      :ok,
+      socket
+      |> assign(params)
+    }
+  end
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
+      <.header>
+        <div class="font-dn text-4xl">
+          <%= to_title_case(@source.title) %>
+        </div>
+      </.header>
+
+      <.back patch={~p"/explore/"}>Back to All Sources</.back>
```

Comment:
navigation done in a way that prefers push_patch() vs push_navigate(). this is prevents full page reloads and keeps things pretty for us.

only drawback is that now, I need to manually trigger scroll ups, since the swapping out of live_components won't default to a "scroll-to-top" behaviour.